src/db/models/dishes/schema.ts

```diff
@@ -42,7 +42,10 @@ export const new_dish_schema = dishes_schema.omit({
 	date_updated: true,
 	ingredients: true,
 	instructions: true,
-	notes: true
+	notes: true,
+	serving_size: true,
+	prep_time: true,
+	cook_time: true
```

Comment:
This is okay and works, but it will be easier to explicitly say which fields SHOULD be on `new_dish_schema` rather than using `.omit()` for the dishes_schema. Because we'd have to update this every time we added/removed from dishes_schema.